pkgs/top-level/all-packages.nix

```diff
@@ -11706,7 +11706,6 @@ in
 
   terra = callPackage ../development/compilers/terra {
     llvmPackages = llvmPackages_6;
```

Comment:
I just tested a build with this swapped to `llvmPackages_10` and it passes all tests. I strongly recommend switching that, because modern LLVM provides Web Assembly, among other advantages. Fundament needs that, for sure. It also provides better optimization passes and a newer JIT model.

Reply:
Done